src/beautifiers/prettydiff.coffee

```diff
@@ -117,6 +117,7 @@ module.exports = class PrettyDiff extends Beautifier
         source: text
         lang: lang
         mode: "beautify"
+        crlf: getDefaultLineEnding() # solved issue 1457
```

Comment:
- [ ] Support custom `eol` from `options`. See https://github.com/Glavin001/atom-beautify/blob/c3e554d9c48ac6880bed8bd9bf3bd8407b855438/src/beautifiers/js-beautify.coffee#L32 for example.